packaging/systemd/cloudstack-agent.service

```diff
@@ -18,8 +18,8 @@
 [Unit]
 Description=CloudStack Agent
 Documentation=http://www.cloudstack.org/
-Requires=libvirtd.service
-After=libvirtd.service
```

Comment:
@wido this needs to be reverted, as the renaming of libvirtd to libvirt-bin happens via the debian packging scripts. This will cause regressions for KVM. If found KVM hosts failing to start.